api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -1389,4 +1390,49 @@ public static void setUseSystemClassLoader(boolean useSystemClassLoader) {
 	public static boolean isUseSystemClassLoader() {
 		return getServiceContext().isUseSystemClassLoader();
 	}
+
+	/**
+	 * Evicts the entity data for a particular entity "instance".
+	 *
+	 * @param sessionFactory
+	 * @since 2.6.0
+	 */
+	public static void evictSingleEntity(SessionFactory sessionFactory, Class<?> entityClass, String uuid) {
+		if(StringUtils.isBlank(uuid)) {
+			evictAllEntities(sessionFactory, entityClass);
```

Comment:
The method says **evictSingleEntity**, in my opinion you should require the uuid rather than evicting all